charts/astronomer/templates/houston/houston-backend-secret.yaml

```diff
@@ -13,9 +13,11 @@ metadata:
     component: {{ template "houston.backendSecret" . }}
 type: Opaque
 data:
-  {{- if .Values.houston.backendSecretConnection }}
+  {{- if .Values.pgbouncer.enabled -}}
+  connection: {{ (printf "postgresql://%s:%s@%s-pgbouncer:%s/%s" .Values.pgbouncer.connection.user .Values.pgbouncer.connection.pass .Release.Name (.Values.ports.pgbouncer | toString) .Values.pgbouncer.connection.db) | b64enc | quote }}
```

Comment:
It's probably not great to require any passwords in the Values file to enable SSL.